odkx-src/spelling_wordlist.txt

```diff
@@ -123,6 +123,7 @@ hotspots
 html
 http
 https
+homescreen
```

Comment:
The term should be home screen (that is what is used in the rest of the documentation) homescreen is not correct. Please remove this addition/reverse this change.